sw/device/silicon_creator/lib/drivers/keymgr.c

```diff
@@ -130,3 +131,139 @@ void keymgr_advance_state(void) {
 rom_error_t keymgr_state_check(keymgr_state_t expected_state) {
   return expected_state_check(expected_state);
 }
+
+/**
+ * Fails if the keymgr is not idle.
+ *
+ * @return OK if the key manager is idle, OTCRYPTO_RECOV_ERR otherwise.
```

Comment:
is `kErrorKeymgrInternal` a `OTCRYPTO_RECOV_ERR`?

Reply:
...nope, I was just not careful enough with copy-pasting from cryptolib. Thanks for the catch!